pixi/backend/api.js

```diff
@@ -19,6 +19,7 @@ const {lint, LintMode} = require('@ampproject/toolbox-linter');
 const cheerio = require('cheerio');
 const log = require('@lib/utils/log')('Pixi API');
 const RateLimitedFetch = require('@lib/utils/rateLimitedFetch');
+const API_VERSION = 0;
```

Comment:
Do we simply want to use a simple interger or more complex version numbers?
I would prefer to start with a 1.
The API here is not likely to change in general, only in the linter results.
Would you update the version number when the linter results change?
That would require an update of the number here, when the linter changes.
Maybe we can be more clever and use two version numbers (or a combined one) where we have the overall API version and the linter version for the data structure, so that there is no need for manual updates when the linter changes.